features/mouse-events.yml

```diff
@@ -0,0 +1,39 @@
+name: Mouse events
+description: Mouse events, such as `click`, `mousedown`, or `mousemove`, fire when users interact with a pointing device such a mouse or trackpad.
+spec: https://w3c.github.io/uievents/#events-mouse-types
+compat_features:
+  # TODO: compute_from list
+  - api.Element.click_event
+  - api.Element.dblclick_event
+  - api.Element.mousedown_event
+  - api.Element.mouseup_event
+  - api.Element.mousemove_event
+  - api.Element.mouseout_event
+  - api.Element.mouseover_event
+  - api.MouseEvent
+  - api.MouseEvent.altKey
+  - api.MouseEvent.button
+  - api.MouseEvent.clientX
+  - api.MouseEvent.clientY
+  - api.MouseEvent.ctrlKey
+  - api.MouseEvent.metaKey
+  - api.MouseEvent.pageX
+  - api.MouseEvent.pageY
+  - api.MouseEvent.relatedTarget
+  - api.MouseEvent.screenX
+  - api.MouseEvent.screenY
+  - api.MouseEvent.shiftKey
```

Comment:
Most of these look good to me based on the spec and looks like everything predating pointer events -> https://www.w3.org/TR/2019/WD-pointerevents3-20191212/#intro